<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>777</width>
    <height>654</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>751</width>
     <height>631</height>
    </rect>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>511</width>
       <height>141</height>
      </rect>
     </property>
     <property name="title">
      <string>Leg 1</string>
     </property>
     <widget class="QSlider" name="x_slider_1">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>40</y>
        <width>281</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="y_slider_1">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>70</y>
        <width>281</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="z_slider_1">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>100</y>
        <width>281</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="x_decrement_1">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>61</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="x_increment_1">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>30</y>
        <width>61</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>&gt;</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="y_decrement_1">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>61</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="y_increment_1">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>60</y>
        <width>61</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>&gt;</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="z_decrement_1">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>61</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="z_increment_1">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>90</y>
        <width>61</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>&gt;</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>X</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Y</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Z</string>
      </property>
     </widget>
     <widget class="QLabel" name="z_Value_1">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>90</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="x_value_1">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>30</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="y_Value_1">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>60</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>511</width>
       <height>141</height>
      </rect>
     </property>
     <property name="title">
      <string>Leg 2</string>
     </property>
     <widget class="QSlider" name="x_slider_3">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>40</y>
        <width>281</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="y_slider_3">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>70</y>
        <width>281</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="z_slider_3">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>100</y>
        <width>281</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="x_decrement_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>61</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="x_increment_3">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>30</y>
        <width>61</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>&gt;</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="y_decrement_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>61</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="y_increment_3">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>60</y>
        <width>61</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>&gt;</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="z_decrement_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>61</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="z_increment_3">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>90</y>
        <width>61</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>&gt;</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>X</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Y</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Z</string>
      </property>
     </widget>
     <widget class="QLabel" name="z_Value_3">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>90</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="x_value_3">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>30</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="y_Value_3">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>60</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>511</width>
       <height>141</height>
      </rect>
     </property>
     <property name="title">
      <string>Leg 3</string>
     </property>
     <widget class="QSlider" name="x_slider_4">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>40</y>
        <width>281</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="y_slider_4">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>70</y>
        <width>281</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="z_slider_4">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>100</y>
        <width>281</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="x_decrement_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>61</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="x_increment_4">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>30</y>
        <width>61</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>&gt;</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="y_decrement_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>61</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="y_increment_4">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>60</y>
        <width>61</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>&gt;</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="z_decrement_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>61</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="z_increment_4">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>90</y>
        <width>61</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>&gt;</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>X</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Y</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Z</string>
      </property>
     </widget>
     <widget class="QLabel" name="z_Value_4">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>90</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="x_value_4">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>30</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="y_Value_4">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>60</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>430</y>
       <width>511</width>
       <height>141</height>
      </rect>
     </property>
     <property name="title">
      <string>Leg 4</string>
     </property>
     <widget class="QSlider" name="x_slider_5">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>40</y>
        <width>281</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="y_slider_5">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>70</y>
        <width>281</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="z_slider_5">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>100</y>
        <width>281</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="x_decrement_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>61</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="x_increment_5">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>30</y>
        <width>61</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>&gt;</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="y_decrement_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>61</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="y_increment_5">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>60</y>
        <width>61</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>&gt;</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="z_decrement_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>61</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="z_increment_5">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>90</y>
        <width>61</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>&gt;</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>X</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Y</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Z</string>
      </property>
     </widget>
     <widget class="QLabel" name="z_Value_5">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>90</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="x_value_5">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>30</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="y_Value_5">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>60</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>30</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>IP Address</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>70</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="cursorPosition">
      <number>0</number>
     </property>
     <property name="placeholderText">
      <string>Port</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>60</y>
       <width>131</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
